requirements.txt

```diff
@@ -92,3 +92,5 @@ retrying>=1.3.3,<2.0
 # License: Apache 2.0
 # Upstream url: https://pypi.org/project/dataclasses/
 dataclasses==0.8; python_version < '3.7'
+google-cloud-bigquery==2.8.0
```

Comment:
could we put this as optional deps in setup.py?

Reply:
Definitely.